hail/python/hailtop/httpx.py

```diff
@@ -104,27 +104,45 @@ def __init__(self,
 
         if timeout is None:
```

Comment:
Merged both:
```
+<<<<<<< HEAD
+        if timeout is None:
+            timeout = aiohttp.ClientTimeout(total=20)
+        if isinstance(timeout, (float, int)):
+            timeout = aiohttp.ClientTimeout(total=timeout)
+||||||| b7bde56d5
+        if timeout is None:
+            timeout = aiohttp.ClientTimeout(total=5)
+        if isinstance(timeout, (float, int)):
+            timeout = aiohttp.ClientTimeout(total=timeout)
+=======
         configuration_of_timeout = configuration_of(ConfigVariable.HTTP_TIMEOUT_IN_SECONDS, timeout, 5)
         del timeout
 
@@ -110,6 +121,7 @@ class ClientSession:
         if isinstance(configuration_of_timeout, (float, int)):
             configuration_of_timeout = aiohttp.ClientTimeout(total=configuration_of_timeout)
         assert isinstance(configuration_of_timeout, aiohttp.ClientTimeout)
+>>>>>>> e68103e03
```

Reply:
Remember that our local change is just to change the default from 5 to 20.

They've refactored this to introduce this `configuration_of_timeout` stuff. So I would use their version, drop the existing four lines, and change the default on the new first line (i.e., the `configuration_of_timeout = configuration_of(…)` line) from 5 to 20.

Reply:
Yes, silly me :-), makes sense.